<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="LabelCapture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>581</width>
      <height>481</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditCamera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpenCloseCamera">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>MODO AUTOMATICO</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelInformationServerClient">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modo: Sin conexi√≥n</string>
    </property>
   </widget>
   <widget class="QLabel" name="LabelCaptureMask">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>581</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
